src/model/position.js

```diff
@@ -245,6 +246,44 @@ export default class Position {
 	}
 
 	/**
+	 * Use forward {@link module:engine/model/treewalker~TreeWalker TreeWalker} to get the farthest position which
```

Comment:
"Uses".

Also, I'm not sure whether mentioning that it uses the TW in the first sentence makes any sense. This is implementation detail.

Finally, "farthest position which matches the callback" is confusing. Is it the furthest position in the document? I guess nope. So perhaps the better comment would be "farthest position which still matches the callback"? Or "last position which still ..."?